Userland/Games/Chess/ChessWidget.cpp

```diff
@@ -548,12 +548,12 @@ ErrorOr<void> ChessWidget::import_pgn(Core::File& file)
     bool recursive_annotation = false;
     bool future_expansion = false;
     Chess::Color turn = Chess::Color::White;
-    DeprecatedString movetext;
+    String movetext;
 
     for (size_t j = i; j < lines.size(); j++)
-        movetext = DeprecatedString::formatted("{}{}", movetext, lines.at(i).to_deprecated_string());
+        movetext = String::formatted("{}{}", movetext, lines.at(i)).release_value_but_fixme_should_propagate_errors();
```

Comment:
I think this can be `TRY()`'d, since the method is `ErrorOr<void>`?

Reply:
@Dexesttp - Thanks for the review - Have just resolved the above